### PR TITLE
Ogtags

### PR DESCRIPTION
**Pull request by crabsinger**

### CODE REVIEW

File: api/og/tags.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  const path = req.headers['x-original-path'] as string;
+
+  if (!path) {
+    return res.status(400).json({ message: 'Missing x-original-path header' });
+  }
+
+  if (path.startsWith('/0x')) {
+    const address = path.substring(1);
```
Comment (crabsinger) [inline api/og/tags.ts:21]: eth addresses?

Comment (CryptoGraffe) [reply api/og/tags.ts:21]: yes

File: api/og/bqimage/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+export const config = {
+  runtime: 'edge',
+};
+
+export default async function handler(req: VercelRequest) {
+  try {
+    const originalUrl = new URL(req.url as string);
+
+    const parts = originalUrl.pathname.split('/');
+    const id = parts[parts.length - 1] ?? 'IDK';
```
Comment (crabsinger) [inline api/og/bqimage/[id].tsx:17]: idk idk

File: cloudflare/workers/ogtags/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+export default {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	async fetch(originalRequest: Request, env: Env): Promise<Response> {
+		// For testing purposes, let us fake the original url
+		const originalUrl = originalRequest.headers.get('X-Original-Url') || new URL(originalRequest.url).searchParams.get('originalUrl');
+
+		const apiURL = env.OG_TAG_API_URL;
+		let request = originalRequest;
+		if (originalUrl) {
```
Comment (crabsinger) [inline cloudflare/workers/ogtags/src/index.ts:21]: test flag or something?

File: api/og/profileimage/[address].tsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+              justifyContent: 'center',
+              justifyItems: 'center',
+            }}
+          >
+            <img
+              alt="avatar"
+              height={200}
+              src={
+                profile.avatar
+                  ? 'https://coordinape-staging.s3.amazonaws.com/' +
```
Comment (crabsinger) [inline api/og/profileimage/[address].tsx:83]: swap to rpod

File: package.json [hunk 5]
```diff
@@ -58,11 +59,13 @@
     "d3-force-3d": "2.3.2",
     "date-fns": "^2.28.0",
     "dedent": "^0.7.0",
+    "escape-html": "^1.0.3",
```
Comment (crabsinger) [inline package.json:62]: please change this to remove this line

**Pull request merged**